third-party/utilities/CMakeLists.txt

```diff
@@ -20,11 +20,14 @@ file(GLOB_RECURSE UTILITIES_HEADER_FILES
     "${UTILITIES_INCLUDE_DIR}/*.h"
 )
 if( BUILD_EASYLOGGINGPP )
-    list( APPEND UTILITIES_HEADER_FILES
-        "${REPO_ROOT}/third-party/easyloggingpp/src/easylogging++.h"
-        )
+    target_sources( ${PROJECT_NAME}
+        PRIVATE "${REPO_ROOT}/third-party/easyloggingpp/src/easylogging++.h" )
 endif()
 target_sources( ${PROJECT_NAME} PRIVATE ${UTILITIES_HEADER_FILES} )
+source_group(
+    TREE ${UTILITIES_INCLUDE_DIR}
```

Comment:
We can keep it as we spoke but honestly I didn't like to break U16 build capability for a CMake flag..
I believe that when we break something we need a good reason..

Reply:
We have much bigger problems than U16.